include/experimental/__p0009_bits/config.hpp

```diff
@@ -184,6 +184,13 @@ static_assert(MDSPAN_IMPL_CPLUSPLUS >= MDSPAN_CXX_STD_14, "mdspan requires C++14
 #  endif
 #endif
 
+#ifndef MDSPAN_IMPL_USE_IF_CONSTEXPR_17
```

Comment:
Do we actually test mdspan with C++14 any more?  If not, then there's no need to have a macro for this feature, and the macro wouldn't be tested anyway.

Reply:
Yes we still have some CI builds (there is at least one user who still uses C++14). We don't guarantee that everything still works with C++14 though and guard against that. For example padded layouts will not work with 14. The MDSpan class does need to so unfortunately we still need this switch

Reply:
Thanks for explaining!  The macro is an implementation detail so it's fine.